gradle/helpers/test-config.gradle

```diff
@@ -12,14 +13,13 @@ task unit (type: Test) {
 task cucumber (type: Test) {
     testLogging { afterSuite { desc, result -> printTestResults(desc,result,'CUKE') } }
     include '**/cucumber/**'
+    testLogging.events = ['standard_out']
 
     // Use for debugging specific Cucumber tests
     // Add the @debug tag above a feature or scenario you want to test
     //systemProperty "cucumber.options", "--tags @debug"
```

Comment:
Couldn't this be done using Gradle's `-D` flag, ie. `gradle cucumber -Dcucumber.options="--tags @debug"`?

Reply:
Yes, it's really just there because I forgot the command a couple of times. It's also convenient since usually when I used it I want it on for an extended period.